conductor/lib/uploader.py

```diff
@@ -219,27 +217,25 @@ def do_work(self, job, thread_int):
 
             self.put_job((path, upload_url))
 
-        ''' make sure we return None, so no message is automatically added to the
-        out_queue '''
+        # make sure we return None, so no message is automatically added to the out_queue
         return None
 
 
-
-'''
-This woker recieves a (filepath: signed_upload_url) pair and performs an upload
-of the specified file to the provided url.
-'''
 class UploadWorker(worker.ThreadWorker):
+    '''
+    This worker receives a (filepath: signed_upload_url) pair and performs an upload
+    of the specified file to the provided url.
+    '''
     def __init__(self, *args, **kwargs):
         worker.ThreadWorker.__init__(self, *args, **kwargs)
         self.chunk_size = 1048576  # 1M
         self.report_size = 10485760  # 10M
         self.api_client = api_client.ApiClient()
 
     def chunked_reader(self, filename):
-        with open(filename, 'rb') as file:
+        with open(filename, 'rb') as _file:
```

Comment:
`_file` could be renamed to `fp` for file pointer (which is a convention I've seen), or something more explicit.